dashboard/app/models/video.rb

```diff
@@ -21,4 +21,47 @@ def self.setup
     end
     check_i18n_names
   end
+
+  def self.youtube_base_url
+    'https://www.youtube.com'
+  end
+
+  def self.youtube_url(code, args={})
```

Comment:
where does code come from? Is it a property of the Video model and if so, should this be an instance method that gets code from self?